src/explore-education-statistics-admin/src/pages/publication/components/PublicationManageTeamAccessTab.tsx

```diff
@@ -43,14 +63,17 @@ const PublicationManageTeamAccessTab = ({ publication, release }: Props) => {
   return (
     <>
       <h2>Update access for release ({release.title})</h2>
-      {!contributors.length ? (
+      {!contributors.length && !pendingInviteEmails.length ? (
```

Comment:
It's usually a good idea to use numeric comparisons on `length` instead relying on a falsy check. 

We typically use `&&` to conditionally render fragments and when combined with falsy `length` checks, this has previously meant that random 0's would appear in the UI when the check is falsy.

As we're using a ternary, it's not such a big deal, but it's probably better to avoid that possibility in the future (maybe due to refactoring).